Source/Public/Get-VSTeamProcess.ps1

```diff
@@ -1,56 +1,46 @@
 function Get-VSTeamProcess {
    [CmdletBinding(DefaultParameterSetName = 'List')]
+   [OutputType([vsteamprocess])]
    param(
-      [Parameter(ParameterSetName = 'List')]
+      [Parameter(ParameterSetName = 'ByName', Position=0)]
+      [ArgumentCompleter([ProcessTemplateCompleter])]
+      [Alias('ProcessName','ProcessTemplate')]
+      $Name = '*',
+
+      [Parameter(DontShow=$true, ParameterSetName = 'List')]

```

Comment:
Why are you setting DontShow here? Why would we want to hide this?

Reply:
See above on the help. 
It looks top and skip are ignored when placed in the URI  (this reference in the docs has airbrushed them out https://docs.microsoft.com/en-us/rest/api/azure/devops/core/Processes/Get?view=azure-devops-rest-5.1 ) 
To avoid breaking any existing scripts the parameters need to remain, but since they will be ignored they're best not shown.

Reply:
I wonder if when that was written it was a copy and paste error. I reviewed the oldest version of the API we support and can't find those anywhere! https://docs.microsoft.com/en-us/azure/devops/integrate/previous-apis/tfs/processes?view=tfs-2017.  So I will remove those and change the version number to 7.x to denote a breaking change.

Reply:
:-) 

In a spirit of gentle teasing (and **not** criticism because you could find I've done exactly the same) but _"was it ever tested and found to work against a live system"._ 

I have found that rather than removing parameters, changing the version and telling people to change their code, it can better in practice to hide the parameter and write a warning when it anyone uses it saying "This is ignored, and may not be supported in future versions".    
Have you seen how annoying it is in Pester 5 that debug in vscode tries to run `invoke-pester -script blah.ps1 ` and pester says "`-script is an unknown switch`"   but `invoke-pester blah.ps1` works so it would be trivial to keep compatibility with a parameter alias.    
Being too  gung-ho about "break this, break that, but it's OK if I change the version number:... if people don't keep up to date and / or flag a maximum version and make sure they load and old one that's _their_ problem" just annoys the user base. 
Microsoft was based on the premise that old stuff worked with new stuff - CP/M recompiled for DOS, DOS ran under Windows, OS/2 and NT,  OS/2 and Win 3.1 also ran on NT, 32bit Windows runs on 64 bit Windows.  So Microsoft veterans like me (Microsoft user in late 70s, 80s and early 90s, Microsoft certified partner mid and late 90s, Microsoft employee 2000-2010) can get a bit rude if people say backwards compatibility is hard :-)  "You don't know you're born ! In my day....  Nurse! "

Reply:
The warning idea sounds good. Letting them know these parameters will be removed in future versions.